scripts/src/generate.ts

```diff
@@ -32,89 +30,68 @@ const run = async () => {
     });
   }
 
+  const outputDir = "./src/generated-method-types/";
```

Comment:
nit: I'm wondering if we will be generating anything else. Would it be beneficial to keep the `generated/` directory and place these files in `generated/method-types/` or something?